FlowCal/io.py

```diff
@@ -1095,25 +1097,29 @@ def amplifier_gain(self, channels=None):
         else:
             return self._amplifier_gain[channels]
 
-    def domain(self, channels=None):
+    def range(self, channels=None):
         """
-        Get the domain of the specified channel(s).
+        Get the range of the specified channel(s).
 
-        The domain is inferred from the $PnR parameter, as
-        ``np.arange($PnR)``. The domain should be transformed along with
-        the data when passed through a transformation function.
+        The range is inferred from the $PnR parameter as ``[0, $PnR - 1]``.
```

Comment:
isn't the range different for floating point? i thought you said elsewhere it was [0, $PnR]